copyright

```diff
@@ -100,6 +100,8 @@ License: CC-BY-SA-4.0
 Files:
  images/land/sky2.jpg
  images/land/sea3.jpg
+ images/land/beach4.jpg
```

Comment:
this conflicts with [L323](https://github.com/endless-sky/endless-sky/blob/dd5dfcacfa9dd484806f598ce65a49d4bbb2cfda/copyright#L323)

Reply:
Notably sky2 and sea3 are also elsewhere in the copyright file, but canyon9 is now.

Reply:
What would be a preferable arrangement?

Reply:
(disclaimer: this is me thinking logically, not based on debian's specifications. in fact, i tried to find any guidelines regarding this case, but debian apparently didn't consider that somebody would want to license the same file twice)

it would be less confusing if the original source was under a wildcard (`images/land/beach*`), and the modification was explicitly stated. We actually have a similar case right in this repo:
```
Files: sounds/*
[...]

Files: sounds/heavy?rocket?hit.wav
```

this case is clear - the wildcard has less "weight" than the explicit statement below it. as it stands with this change, however, both have equal weight. without looking at this PR, it's impossible to tell for sure which file is the original and which one is modified. (There is a wildcard, butas Amazinite pointed out below, your change should have the same wildcard, so it's negligible)

imo if you want to mention the original source, do it like this:
```
Files: sounds/*
Copyright: Various
License: public-domain
Based on public domain sounds taken from freesound.org.
```